jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -135,6 +150,15 @@ public PermissionConfiguration() {}
     @Nullable @Parameter private String credHelper;
 
     @Parameter private FromAuthConfiguration auth = new FromAuthConfiguration();
+
+    @Parameter private List<PlatformsConfiguration> platforms;
+
+    /** Configuration for {@code platforms} parameter. */
```

Comment:
This comment seems out of place?

Reply:
Right maybe change it to `/** Constructor for defaults . **/` or something?

Reply:
Done